.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+indent_size = 2
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[Makefile]
+indent_style = tab
+indent_size = 4
+
+[*.xml]
+indent_size = 4
+
+[*.yml,*.yaml]
+indent_size = 4
```

Comment:
please end all files with a newline

Reply:
format command is run on all scripts, I can't manually add new lines int he code.

Reply:
as far as I see from `.editorconfig:9` every file should end with a newline

Reply:
like I said I run npm run format on every module and pushed it as such.,

Reply:
please adjust all the files to the format defined in `.editorconfig`

Reply:
what do you want me to do here, I run npm run format and lint not sure what else I should be doing here.

Reply:
new line is there now